website_wow/static/tests/js/website_wow.editor.tour.js

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.tour_editor', function (require) {
+    'use strict';
+
+    var tour = require("web_tour.tour");
+    var base = require("web_editor.base");
+    var snippet_editor = require('web_editor.snippet.editor');
+
+    require('website_wow.editor');
+    require('website_wow.view');
+
+    tour.register("website_wow_editor", {
+        url: '/page/contactus?debug=assets&enable_editor=1',
```

Comment:
`debug=assets` should not be required, although it can help debugging these problems...